.circleci/config.yml

```diff
@@ -230,6 +249,12 @@ workflows:
           requires:
             - request-staging-approval
 
+      - acceptance-tests:
```

Comment:
This is just an approval, it won't actually do anything. You need another task that requires this one and runs your acceptance tests job 🙂

Reply:
just updated the script